docs/topics/cache.txt

```diff
@@ -363,6 +363,16 @@ user ``apache``, make sure the directory ``/var/tmp/django_cache`` exists and
 is readable and writable by the user ``apache``, or that it can be created by
 the user ``apache``.
 
+.. warning:: 
+
+    The filesystem cache is intended as an easy way to test caching, not as a 
+    serious caching strategy. It is not designed to handle a large number of 
+    files.    
+
+    If you do need to use filesystem caching for a large number of files then
+    it is recommended that you subclass ``FileBasedCache`` and override the 
+    ``_cull`` function.
```

Comment:
I would say _"improve culling strategy"_ instead of _"override the ``_cull`` function"_.

Reply:
What about both? Or link docs? It would be a bit sad to have every new person that gets this message spend 10-30 minutes on research.

Reply:
I couldn't find any docs about that - I figured out what to do by reading the code.   I can make this a bit more verbose. Would it be better if I didn't mention the _cull function?